jvm/src/test/kotlin/app/cash/trifle/CertificateRequestTest.kt

```diff
@@ -0,0 +1,120 @@
+package app.cash.trifle
+
+import app.cash.trifle.internal.TrifleAlgorithmIdentifier
+import app.cash.trifle.internal.signers.TinkContentSigner
+import app.cash.trifle.internal.signers.TrifleContentSigner
+import app.cash.trifle.internal.util.TestFixtures
+import com.google.crypto.tink.signature.SignatureConfig
+import okio.ByteString.Companion.toByteString
+import org.bouncycastle.asn1.x500.X500Name
+import org.bouncycastle.asn1.x509.AlgorithmIdentifier
+import org.bouncycastle.asn1.x509.SubjectPublicKeyInfo
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import org.bouncycastle.operator.DefaultSignatureNameFinder
+import org.bouncycastle.pkcs.PKCS10CertificationRequestBuilder
+import org.bouncycastle.pkcs.PKCSException
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.OutputStream
+import java.security.KeyPair
+import java.security.KeyPairGenerator
+import java.security.SecureRandom
+import java.security.Security
+import java.security.Signature
+import java.security.spec.ECGenParameterSpec
+
+internal class CertificateRequestTest {
+
+  @BeforeEach
+  fun setUp() {
+    SignatureConfig.register()
```

Comment:
do we need this line?

Reply:
no.  leftover from some tink stuff I did and forgot to erase